XamlControlsGallery/ControlPages/ImplicitTransitionPage.xaml.cs

```diff
@@ -137,6 +137,15 @@ private void BackgroundButton_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        private double EnsureValueIsNumber(NumberBox numberBox)
```

Comment:
Could this return a float instead? Then we wouldn't need to cast above.

Reply:
Good point, fixed now.